portfolio/src/main/java/com/google/sps/data/UserManager.java

```diff
@@ -0,0 +1,16 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+
+public class UserManager {
+    public static String getCurrentUserId() {
+        UserService userService = UserServiceFactory.getUserService();
+
+        if (!userService.isUserLoggedIn()) {
+            return null;
+        }
+
+        return userService.getCurrentUser().getUserId();
```

Comment:
Optionally you could return a ternary here.

`return userService.isUserLoggedIn() ? userService.getCurrentUser().getUserId() : null;`